### ISSUE TITLE
Some UX Feedback

### ISSUE THREAD

**Issue by slifty** [BadIdeaFactory/the-emoji-compass#89]
From @reefdog:

1. I saw the red/gray distinction as being that the gray ones were meant to be the draggable ones and the red one would be the answer. (Since the language sets me up for a 3:1 distinction, and that was the most immediately evident 3:1 distinction; now I see the length difference as well but I didn’t at first.)

2. I don’t think the answer one should ever share a color with the manually-movable ones. So like, gray for the inactive draggable, red for the active draggable, and gold or something for the auto-moving one would be a good distinction. (edited) 
3. The gray is too similar in darkness to the red; I think they need to have more contrast from each other. (Turn your screen into b/w and they’re totally indistinguishable; they shouldn’t be.)

**Comment by slifty** [comment]
One thought: maybe adding a little wiggle to the dial you have to interact with next would be helpful (not a constant wiggle, but a ".. hey you!!" kind of repeating wiggle)

**Comment by reefdog** [comment]
Yeah, it seems there's a not easily-intuited required sequence: first this one, then this one, then this one; and once one is set, it's set.

I think it would help for that sequence to be apparent from the start, and I think that's going to require more visual differentiation of state and order.

So at start, the first one is in a state of unmoved movability and the other are in a state of unmoved unmovability. After moving the first one, it is now moved unmovability, while the second one is unmoved movability and the last one is unmoved unmovability. After moving the second one, the first two are in a state of moved unmovability and the third is unmoved movability. After moving the third, they're all three in a state of moved unmoveability.

The answer needle, too, could benefit from a distinction of states between "resting" and "active/answering".

I think I would use color to distinguish "moved" vs "unmoved" and opacity/brightness to distinguish "movable" form "unmovable". So unmoved-movable could be full-opacity red; unmoved-unmovable could be half-opacity gray; moved-unmovable could be half-opacity red. (The answer needle could be half-opacity gold while resting and full-opacity gold while active/answering.) Thoughts?

**Comment by chriszs** [comment]
If the hands are set one at a time I wonder if a simple fix would be to make them movable by dragging anywhere on the dial instead of directly on the hand.

**Comment by maboa** [comment]
This might be a separate issue and very nitpicky but I'll mention it anyway (since we're talking about related things) - regarding the animation we have when the grey hand settles into place, I think it could be improved with the appropriate easing equation.
